test/unit/org/apache/cassandra/io/sstable/LegacySSTableTest.java

```diff
@@ -598,10 +598,12 @@ private static void loadLegacyTable(String tablePattern, String legacyVersion) t
      * <p>
      * Run this test alone (e.g. from your IDE) when a new version is introduced or format changed
      * during development. I.e. remove the {@code @Ignore} annotation temporarily.
+     *
+     * *************RUN THIS TEST ALONE to avoid unwanted compactions, flushes, etc to interfere
```

Comment:
Does this mean that it should be moved to a separate class?

Reply:
I have torn thoughts here: being in the same class sort of makes sense so you don't forget about it. But it's too easy to mess things around as this _has_ to be ran alone :shrug:

Reply:
We can decide between putting it on a separate class or making it work on the same one on a separate ticket. If anything, I would make the comment (which is pretty clear) a TODO `@Ignore` annotation:
```java
@Ignore // TODO: Currently this test needs to be alone to avoid unwanted compactions, flushes, etc to interfere
```

Reply:
I think adding the comment should do the trick. I would have done it for me.

Reply:
Not a big deal, but I think it's clearer to have the comment next to the `@Ignore` tag, since it explains that tag. The `TODO` prefix is highlighted and shown in code analyzers by some IDEs, which might be useful to remind us that at some point we have to fix the test (separate ticket, etc.).

Reply:
It is so, I haven't pushed my latest changes yet.